hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/cluster/SparkExecuteClusteringCommitActionExecutor.java

```diff
@@ -209,7 +209,7 @@ protected String getCommitActionType() {
               .build();
 
           recordIterators.add(HoodieFileSliceReader.getFileSliceReader(baseFileReader, scanner, readerSchema,
-              table.getMetaClient().getTableConfig().getPayloadClass()));
+              table.getMetaClient().getTableConfig().getPayloadClass(), table.getMetaClient().getTableConfig().getPreCombineField()));
```

Comment:
pull out `table.getMetaClient().getTableConfig()` into a variable ? for ease of reading?